zebra-node-services/src/mempool.rs

```diff
@@ -58,6 +58,11 @@ pub enum Request {
     /// The transaction downloader checks for duplicates across IDs and transactions.
     Queue(Vec<Gossip>),
 
+    /// Queue a list of unmined transactions submitted via the `sendrawtransaction` RPC method,
+    /// and wait for them to be verified, rejected, or for their verification task to be cancelled if
+    /// the transaction id is mined into the best chain.
+    QueueRpc(Vec<UnminedTx>),
```

Comment:
It's probably best to return the verification error for all `Queue` requests, (or the `oneshot::Receiver`s so the inbound service can still respond to peers quickly) so it's easier to link transactions to peers for scoring misbehaving peers. Typed errors will also be helpful there, so it would be better to move `TransactionDownloadVerifyError` to `zebra-node-services` than to revert the changes for making it clone-able.